--- coq-8.4pl2/lib/envars.ml	2013-06-12 12:43:35.916131365 +0200
+++ coq-8.4pl2/lib/envars.ml	2013-06-12 12:47:41.276128920 +0200
@@ -32,15 +32,18 @@
 
 let guess_coqlib () =
   let file = "states/initial.coq" in
-    reldir (if Coq_config.arch = "win32" then ["lib"] else ["lib";"coq"]) file
-      (fun () ->
-        let coqlib = match Coq_config.coqlib with
+  (* First try to use the information from build time *)
+  let coqlib = match Coq_config.coqlib with
           | Some coqlib -> coqlib
           | None -> coqroot
-        in
-        if Sys.file_exists (Filename.concat coqlib file)
-        then coqlib
-        else Util.error "cannot guess a path for Coq libraries; please use -coqlib option")
+  in
+    if Sys.file_exists (Filename.concat coqlib file)
+    then coqlib
+    else
+    (* else search the usual directories *)
+    reldir (if Coq_config.arch = "win32" then ["lib"] else ["lib";"coq"]) file
+      (fun () ->
+        Util.error "cannot guess a path for Coq libraries; please use -coqlib option")
 
 let coqlib () =
   if !Flags.coqlib_spec then !Flags.coqlib else
